tuist/tuist
# Ensuring Library search paths and Swift import paths are set

Resolves https://github.com/tuist/tuist/issues/256

### Short description 📝

A few additional build settings are needed when linking against pre-built static libraries

- `LIBRARY_SEARCH_PATHS` needs to point to the directory containing the `.a` file
- `SWIFT_INCLUDE_PATHS` need to point to the directory containing the `.swiftmodule` file

### Solution 📦

> Describe the solution you came up with and the reasons that led you to that solution. If you thought about other solutions don't forget about mentioning them.

### Implementation 👩‍💻👨‍💻

- [x] Update link generator to include missing settings 

### Test Plan ⚒

Test Plan:

- Generate `fixtures/ios_app_with_static_libraries`
- Verify the application builds
- Verify the `LIBRARY_SEARCH_PATHS` and `SWIFT_INCLUDE_PATHS` settings are set on target `A`
